### PR TITLE
#184948240 Setup Redux 

### PR DESCRIPTION
**Pull request by AbdulKhaliq59**
#### What does this PR do?
This PR setting up Redux functionality
#### Description of Task to be completed?
 - [x] `Installing redux`
 - [x] `setting up Store`
 - [x] `Setting up Reducer`
#### How should this be manually tested?
- run `npm run dev`
#### Any background context you want to provide?
- `Initial React project`
#### What are the relevant pivotal tracker/Trello stories?
Story ID: [#184948240](https://www.pivotaltracker.com/story/show/184948240)
#### Screenshots (if appropriate)
N/A
#### Questions:

### CODE REVIEW

File: src/redux/rootReducer.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from "redux";
+
+const rootReducer = combineReducers({
+// Set up your reduce here
+});
```
Comment (Emile-SHUMBUSHO) [inline src/redux/rootReducer.js:5]: Well done but I would like to have some simple reducer, just to test the setup functionality.

**Review by GasanaJr — CHANGES_REQUESTED** [review]
rebase on develop, you have some conflicting files

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Comment by Emile-SHUMBUSHO** [comment]
LGTM except this failing test from circleCI and semaphoric

**Review by GasanaJr — APPROVED** [review]

**Review by khevin2 — APPROVED** [review]

**Review by nshutijonathan — APPROVED** [review]
LGTM

**Pull request merged**